tools/FlatFlowMigrationCli/Operations/MigrateOperation.cs

```diff
@@ -94,6 +106,15 @@ public async Task<int> RunAsync()
             }
         }
 
+        vmrRepositories = vmrRepositories
+            // NuGet will be handled separately (https://github.com/dotnet/arcade-services/issues/4618)
```

Comment:
nit: maybe just add these in a list and do something like `!list.Contains(r.Mapping.Name)`